lib/constants.json

```diff
@@ -61,10 +61,10 @@
         360559640,
         453559640,
         569809640,
-        2147483647
+        999999999999999999999999999999999999999999999999999999
```

Comment:
Ideally, this would be Infinity but for some reason, the constants file is JSON so that is not possible.